src/main/java/com/myhome/domain/HouseHistory.java

```diff
@@ -0,0 +1,30 @@
+package com.myhome.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import java.util.Date;
+
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class HouseHistory  extends BaseEntity {
+
+    @Column(unique = false, nullable = false)
+    private String houseId;
+
+    @Column(nullable = false, unique = false)
+    private String memberId;
```

Comment:
There is no hibernate relationsips with `HouseMember` and `CommunityHouse` entities. Maybe they don't need it, but I'm not quite sure. What do you think?

Reply:
@DDmit04 You are right

Reply:
So what do you suggest? I don't have to give any relation right?
Or should I?

Reply:
You should